docs/quickstart/php.md

```diff
@@ -71,6 +71,10 @@ $ cd Protobuf-PHP
 $ rake pear:package version=1.0
 $ [sudo] pear install Protobuf-1.0.tgz
 ```
+> maybe you should install 'ronn' if the command not found:
+> ```sh
+> $ gem install ronn
```

Comment:
While we are at it, can you add `rake` as well? So `gem install rake ronn`. Thanks.